invenio_communities/utils.py

```diff
@@ -99,13 +100,21 @@ def save_and_validate_logo(logo_stream, logo_filename, community_id):
     cfg = current_app.config
 
     logos_bucket_id = cfg['COMMUNITIES_BUCKET_UUID']
+    logo_max_size = cfg['COMMUNITIES_LOGO_MAX_SIZE']
     logos_bucket = Bucket.query.get(logos_bucket_id)
     ext = os.path.splitext(logo_filename)[1]
     ext = ext[1:] if ext.startswith('.') else ext
 
+    logo_stream.seek(SEEK_SET, SEEK_END)  # Seek from beginning to end
```

Comment:
This is safe, since MAX_CONTENT_LENGTH will not allow anything more than 16MiB by this point anyway.